static/js/containers/pages/DashboardPage.js

```diff
@@ -22,9 +23,16 @@ type DashboardPageProps = {
 }
 
 export class DashboardPage extends React.Component<DashboardPageProps, void> {
+  now = moment()
+  isLinkableCourseRun = ({ run }: RunEnrollment): boolean =>
+    !R.isNil(run.courseware_url) &&
+    !R.isNil(run.start_date) &&
+    moment(run.start_date).isBefore(this.now) &&
+    (R.isNil(run.end_date) || moment(run.end_date).isAfter(this.now))
```

Comment:
This can be cleaned up an moved to an API.

```es6
// static/js/lib/coursesApi.js (new file)
export const isLinkableCourseRun = (run: CourseRun, dtNow?: Moment): boolean => {
  const now = dtNow || moment()
  return {
    notNil(run.courseware_url) &&
    notNil(run.start_date) &&
    moment(run.start_date).isBefore(now) &&
    (isNil(run.end_date) || moment(run.end_date).isAfter(now))
  }
}

// DashboardPage.js
const title = isLinkableCourseRun(enrollment.run) ? () : ()
```

Once you do that, you can test this function in isolation, then mock the function for the `DashboardPage.js` tests